onnxruntime/python/tools/symbolic_shape_infer.py

```diff
@@ -1600,6 +1601,50 @@ def _infer_ReduceSum(self, node):  # noqa: N802
                     )
                 )
 
+    def _infer_ReduceMean(self, node):  # noqa: N802
+        keep_dims = get_attribute(node, "keepdims", 1)
+        opset = get_opset(self.out_mp_)
+
+        if opset >= 13 and len(node.input) > 1:
```

Comment:
ReduceMean-13 still uses attribute. Maybe just check whether there is two inputs, and not check opset version so that it is more general for Reduce* op.